ginga/gw/Desktop.py

```diff
@@ -244,7 +245,7 @@ def _close_page(self, ws):
         if num_children == 0:
             del self.workspace[ws.name]
             root = ws.root
-            bnch.root = None
+            #bnch.root = None  # TODO: Undefined bnch!
```

Comment:
`bnch` is undefined in this method, and I can't figure out where it is supposed to come from, so I commented this.